### PR TITLE
Issue #1104 | Update user guide to include information about sitemaps and robots.txt for multistore configuration

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/merchdocs/issues/1104

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/sitemap-xml.html

## Links to Magento source code or PRs

- https://developers.google.com/search/docs/advanced/sitemaps/large-sitemaps?hl=en&visit_id=637564147951582653-2730712577&rd=1
- https://developers.google.com/search/docs/advanced/robots/intro
- http://httpd.apache.org/docs/2.2/howto/htaccess.html

whatsnew
Updated [Using a Site Map](https://docs.magento.com/user-guide/marketing/sitemap-xml.html) with a new section about managing sitemaps and robots.txt for multiple websites.

### CODE REVIEW

File: src/marketing/sitemap-xml.md [hunk 1]
```diff
@@ -61,6 +61,34 @@ For more information, see the instructions for the specific search engine. Here
...
+To create sitemaps for Magento multistore instances do the following:
+
+1. Create a new folder called sitemaps at the root of your website, then create subfolders for each domain:
+
+    /sitemaps/domain_1/
+    /sitemaps/domain_2/
+
+1. On the Admin menu, go to _Marketing > SEO & Search > Site Map_.
+
+1. Create or edit the sitemap listings for each store and set the “path to sitemap” field to be the path you created for the store:
```
Comment (leslietilling) [inline src/marketing/sitemap-xml.md:77]:
```suggestion
1. Create or edit the sitemap listings for each store and set the **Path** to the one you created for the store:
```

File: src/marketing/sitemap-xml.md [hunk 2]
```diff
@@ -61,6 +61,34 @@ For more information, see the instructions for the specific search engine. Here
...
+
+    /sitemaps/domain_1/
+    /sitemaps/domain_2/
+
+1. Update your robots.txt file. To make sure that the search engine spiders are properly directed to the new sitemaps. You may need to update or create the robots.txt file. Edit the robots.txt file and add the following lines at the top.
```
Comment (leslietilling) [inline src/marketing/sitemap-xml.md:82]:
```suggestion
1. If needed, update your robots.txt file.

   To make sure that the search engine spiders are properly directed to the new sitemaps, you might need to update or create the robots.txt file and add the following lines at the top.
```

File: src/marketing/sitemap-xml.md [hunk 3]
```diff
@@ -61,6 +61,34 @@ For more information, see the instructions for the specific search engine. Here
...
+
+Website Sitemap
+
+    Sitemap: http://www.domain_1.com/sitemaps/domain_1/sitemap.xml
+    Sitemap: http://www.domain_2.com/sitemaps/domain_2/sitemap.xml
+
+{:.bs-callout-info}
+If your site uses the [Apache](https://devdocs.magento.com/guides/v2.4/install-gde/prereq/apache.html) web server engine, you should update the [.htaccess](http://httpd.apache.org/docs/current/howto/htaccess.html) file in the root of your website to direct any other sitemap requests to the proper place.
+
 ## Column Descriptions
```
Comment (leslietilling) [inline src/marketing/sitemap-xml.md:92]:
```suggestion
## Column descriptions
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

**Comment by EvheniiaKolkhiieva** [comment]
Thank you @leslietilling for reviewing!
I have implemented your suggestions, could you please approve this PR?

File: src/marketing/sitemap-xml.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/marketing/sitemap-xml.md]: Done ✅

File: src/marketing/sitemap-xml.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/marketing/sitemap-xml.md]: Done ✅

File: src/marketing/sitemap-xml.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/marketing/sitemap-xml.md]: Done ✅

File: src/marketing/sitemap-xml.md [hunk 4]
```diff
@@ -61,6 +61,34 @@ For more information, see the instructions for the specific search engine. Here
...
+If you have multiple websites, you can simplify the process of creating and submitting sitemaps by [creating]({% link marketing/sitemap-xml-configure.md %}) one or more sitemaps that include URLs for all your verified stores, and saving the sitemap(s) to a single location. All sites must be verified in [Search Console](https://support.google.com/webmasters/answer/7451001).
+
+To create sitemaps for Magento multistore instances do the following:
+
+1. Create a new folder called sitemaps at the root of your website, then create subfolders for each domain:
+
+    /sitemaps/domain_1/
+    /sitemaps/domain_2/
+
+1. On the Admin menu, go to _Marketing > SEO & Search > Site Map_.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/marketing/sitemap-xml.md]: Done ✅

File: src/marketing/sitemap-xml.md [hunk 5]
```diff
@@ -61,6 +61,34 @@ For more information, see the instructions for the specific search engine. Here
 
 You can now restore either the original (default) restrictions.
 
+## Manage sitemaps and robots.txt for multiple websites
+
+If you have multiple websites, you can simplify the process of creating and submitting sitemaps by [creating]({% link marketing/sitemap-xml-configure.md %}) one or more sitemaps that include URLs for all your verified stores, and saving the sitemap(s) to a single location. All sites must be verified in [Search Console](https://support.google.com/webmasters/answer/7451001).
+
+To create sitemaps for Magento multistore instances do the following:
+
+1. Create a new folder called sitemaps at the root of your website, then create subfolders for each domain:
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/marketing/sitemap-xml.md]: Done ✅

File: src/marketing/sitemap-xml.md [hunk 6]
```diff
@@ -61,6 +61,34 @@ For more information, see the instructions for the specific search engine. Here
 
 You can now restore either the original (default) restrictions.
 
+## Manage sitemaps and robots.txt for multiple websites
+
+If you have multiple websites, you can simplify the process of creating and submitting sitemaps by [creating]({% link marketing/sitemap-xml-configure.md %}) one or more sitemaps that include URLs for all your verified stores, and saving the sitemap(s) to a single location. All sites must be verified in [Search Console](https://support.google.com/webmasters/answer/7451001).
+
+To create sitemaps for Magento multistore instances do the following:
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/marketing/sitemap-xml.md]: Done ✅

File: src/marketing/sitemap-xml.md [hunk 7]
```diff
@@ -61,6 +61,34 @@ For more information, see the instructions for the specific search engine. Here
 
 You can now restore either the original (default) restrictions.
 
+## Manage sitemaps and robots.txt for multiple websites
+
+If you have multiple websites, you can simplify the process of creating and submitting sitemaps by [creating]({% link marketing/sitemap-xml-configure.md %}) one or more sitemaps that include URLs for all your verified stores, and saving the sitemap(s) to a single location. All sites must be verified in [Search Console](https://support.google.com/webmasters/answer/7451001).
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/marketing/sitemap-xml.md]: Done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**